exercises/practice/leap/.approaches/if-expression/content.md

```diff
@@ -0,0 +1,57 @@
+# If expresion
+
+```elm
+isLeapYear : Int -> Bool
+isLeapYear year =
+  let
+    divisbleBy number = 
+      modBy number year == 0 
+  in
+    if divisbleBy(400) then
+      return True
```

Comment:
`return`? 
Haha, the copy/paste is strong with this one :)
Where did you take the code from? Do they have the `divisble` typo as well?